rlp-derive/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "rlp-derive"
-version = "0.1.0"
+version = "0.2.0"
```

Comment:
why breaking change in the `derive crate` but not in the rlp?

Reply:
ok I see it was MSRV bump

Reply:
It is a breaking change for `rlp` as well. I've accidentally published `rlp` 0.6.0 with `rlp-derive` versioned at 0.1.0 and then yanked that version. So it will go from 0.5.2 to 0.6.1 (already published).

Reply:
ok, cool missed that

Reply:
@ordian looks like I forgot to bump impl-codec as well 🙈 
it fails the build when I try to build https://github.com/paritytech/polkadot-sdk/pull/5673